README.md

```diff
@@ -1,124 +1,45 @@
-
-Make sure to include installation of `pip` as well, as it will be used for downloading packages.
-
-### Python Packages
-You may want to set up virtual environment on your machine to void 
-package conflicts. But this is optional.
-
-If you are using anaconda, please follow the following step
-<br/>`conda create -n yourenvname python=x.x anaconda`
-<br/>`conda activate yourenvname`
-<br/>`conda install -n yourenvname [package]`
-<br/>`conda deactivate`
-
-The packages needed to run the scripts provided in this guide can be downloaded and installed via:
-
-`pip install -r /<...>/cs2102-python-flask-setup-guide/requirements.txt`
-<br/>or
-<br/>`conda install --file requirements.txt`
-
-Replace `<...>` with the corresponding path to `cs2102-python-flask-setup-guide` folder on your computer.
-
-Here is a list of flask-related packages and its main usages:
-* flask
-    * This package handles routing, request handling and HTML template rendering.
-* flask-login
-    * This package provides login-related utility functions.
-* flask-sqlalchemy
-    * This package provides ORM classes that connect the Flask App to database,
-    and thus allow you to execute SQL queries in your App.
-* flask-wtf
-    * This package handles form submission and pass user-entered values into Python objects.
-    It also provides other utilities such as validation checks on user-entered fields.
-
-### Pycharm (optional)
-Pycharm is a popular IDE for Python.
-If you have ticked IDLE when installing Python, you should already have IDLE, which is a simple Python IDE for you to write and run your codes.
-However, IDLE is quite minimalist and can be hard to debug if you intend to scale up your project.
-Also, you will need to deal with other file types (html, css etc) so using Pycharm can provide you a centralised platform to view everything in one place.
-
-## Sample Web App Walk-through
-
-This should be the directory structure of the files:
-* cs2102-python-flask-setup-guide/
-    * FlaskApp/
-        * static/
-            * css/
-            * fonts/
-            * images/
-            * js/
-        * templates/
-            * index.html
-        * app.py
-        * models.py
-        * views.py
-    * init.sql
-    * README.md
-    * requirements.txt
-
-### Change web app and database configurations
-Open `FlaskApp/app.py` and locate the following lines of codes:
-
-```
+1. Clone the files from its github repo [here](https://github.com/DriverUncle/TumpangMePlease.git). Clone the repo to your desired directory.
+2. Open a terminal, set the directory to the cloned repo, then run the following command to install the dependencies: 
+   -  for pip: `pip install -r requirements.txt`
+   -  for conda: `conda install --file requirements.txt`
+3. In the terminal,  launch PostgreSQL, and load the data from `/init.sql` into the database.
+4. In the file `FlaskApp/app.py`, ensure that the login details match the existing ones in your PostgreSQL.
+```python
```

Comment:
I think might need to highlight some of the main features (e.g. login, register, bid, list-advertisement etc) on a brief level.